src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub const DEFAULT_WS_ADDR: &str = "ws://0.0.0.0:8900";
 #[from_env]
 pub const DEFAULT_TX_MAX_RETRIES: u16 = 1;
 #[from_env]
-pub const DEFAULT_TX_BATCH_SIZE: usize = 128;
+pub const DEFAULT_TX_BATCH_SIZE: usize = 1 << 5;
```

Comment:
why are you stuck on 1<<5 , 128 value is more lisible and 1<<5 is 32 not 128.

Reply:
Yeah as we discussed we don't want a higher batch size.

Reply:
Revert this back